Source/HTTP/httpcall.cpp

```diff
@@ -123,7 +123,7 @@ HRESULT perform_http_call(
                     {
                         try
                         {
-                            performFunc(data->async, call);
+                            performFunc(data->async, call, httpSingleton->m_platformContext.get());
```

Comment:
>httpSingleton->m_platformContext.get() [](start = 59, length = 38)

since you aren't allowing client to override the context, anybody that has set a custom perform function will be given a "random" pointer here they don't know what to do with